prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Absence {
+  id                  Int      @id @default(autoincrement())
+  lessonDate          DateTime
+  subject             Subject
+  lessonPlan          String?
+  reasonOfAbsence     String
+  numberOfStudents    Int
+  absentTeacherId     Int
+  substituteTeacherId Int?
+  locationId          Int
+  absentTeacher       User     @relation("absentTeacher", fields: [absentTeacherId], references: [id])
+  substituteTeacher   User?    @relation("substituteTeacher", fields: [substituteTeacherId], references: [id])
+  location            Location @relation(fields: [locationId], references: [id])
+}
+
+model User {
+  id          Int       @id @default(autoincrement())
+  authId      String
+  email       String    @unique
+  firstName   String
+  lastName    String
+  role        Role      @default(TEACHER)
+  status      Status    @default(INVITED)
+  absences    Absence[] @relation("absentTeacher")
+  substitutes Absence[] @relation("substituteTeacher")
+}
+
+model Location {
+  id           Int    @id @default(autoincrement())
+  name         String
+  abbreviation String
+  absences     Absence[]
+}
+
+model MailingList {
+  id           Int    @id @default(autoincrement())
+  name         String
+  emails       String[]
+}
```

Comment:
This is what I got when running `npx prisma format` locally. But I would suggest re-running it on your end as well.

```suggestion
model Location {
  id           Int       @id @default(autoincrement())
  name         String
  abbreviation String
  absences     Absence[]
}

model MailingList {
  id     Int      @id @default(autoincrement())
  name   String
  emails String[]
}
```